tests/unit/job/test_extract.py

```diff
@@ -37,6 +37,7 @@ def test_ctor_w_properties(self):
 
     def test_to_api_repr(self):
         from google.cloud.bigquery import job
+        import json
```

Comment:
Nit: No reason to do this import inside the test as it's a built-in. Let's put this at the top of the module.